addons/website_quote/models/sale_order.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import datetime
+import uuid
+
+from openerp import api, fields, models
+
+import openerp.addons.decimal_precision as dp
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+    _description = "Sales Order Line"
+
+    website_description = fields.Html(string='Line Description', compute='_compute_website_description', store=True)
+    option_line_id = fields.One2many('sale.order.option', 'line_id', string='Optional Products Lines')
+
+    @api.one
+    def _compute_website_description(self):
```

Comment:
Previously it is only HTML field and it is called on updating line.Previously behavior is same.So i think no need of @api.depend.